### PR TITLE
Singup passphrase screen

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers 
@cjb this has the user card you can use

### CODE REVIEW

File: shared/common-adapters/avatar.js.flow [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import React, {Component} from 'react'
 
 export type Props = {
-  size: number,
+  size: 180 | 110 | 75 | 60 | 45,
```
Comment (chrisnojima) [inline shared/common-adapters/avatar.js.flow]: we only allow certain sizes

File: shared/common-adapters/user-card.desktop.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+// @flow
+
+import React, {Component} from 'react'
```
Comment (chrisnojima) [inline shared/common-adapters/user-card.desktop.js]: This is used in provision / signup a bunch of times

File: shared/update/confirm.js [hunk 3]
```diff
@@ -6,7 +6,6 @@ import {Snackbar} from 'material-ui'
 import {Header, Text, Button, Checkbox, Icon, Terminal} from '../common-adapters'
 import {clipboard} from 'electron'
 import marked from 'marked'
-import {autoResize} from '../../desktop/renderer/remote-component-helper'
```
Comment (chrisnojima) [inline shared/update/confirm.js]: eslint cleanup, please make sure to run eslint

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

**Comment by cjb** [comment]
@chrisnojima :thumbsup:, you'll want to `s/globalColorsDZ2/globalColors/g` and `s/ dz2 //g`

**Pull request merged**